package.json

```diff
@@ -22,10 +22,12 @@
   },
   "devDependencies": {
     "@guardian/eslint-config-typescript": "^0.5.0",
+    "@types/aws-lambda": "^8.10.72",
     "@types/jest": "^26.0.20",
     "@types/node": "14.14.32",
     "@typescript-eslint/eslint-plugin": "^4.16.1",
     "@typescript-eslint/parser": "^4.16.1",
+    "aws-sdk": "^2.866.0",
```

Comment:
I think that you could use [`v3` of this sdk](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/index.html). I had a few problems migrating a project to it recently, but unless you are aware of any known issues with the functionality you're using it could be worth a try?

Reply:
Good shout! I've tried it and looks fine, thanks for the heads up about this.

Reply:
Sadly, I've had to revert to v2 of the library since an AWS Node Lambda [only has v2 of the library available](https://docs.aws.amazon.com/lambda/latest/dg/lambda-nodejs.html). If and when this changes we can definitely upgrade though, but not sure we can make this work with v3 at the moment.